test/workerd/tests.mjs

```diff
@@ -33,17 +33,32 @@ export const url_parse = {
 
 // --- node:buffer
 
-export const buffer_implements = {
+export const unenv_polyfills_buffer = {
   async test() {
-    const encoder = new TextEncoder();
     const buffer = await import("unenv/runtime/node/buffer");
     const Buffer = buffer.Buffer;
+    assert.strictEqual(typeof buffer.isAscii, "function");
+    assert.strictEqual(typeof buffer.isUtf8, "function");
+    assert.strictEqual(buffer.btoa("hello"), "aGVsbG8=");
+    assert.strictEqual(buffer.atob("aGVsbG8="), "hello");
+    assert.strictEqual(typeof buffer.transcode, "function");
+    assert.strictEqual(typeof buffer.File, "function");
+    assert.strictEqual(typeof buffer.Blob, "function");
+    assert.strictEqual(typeof buffer.INSPECT_MAX_BYTES, "number");
+    assert.strictEqual(typeof buffer.resolveObjectURL, "function");
+    assert.strictEqual(typeof Buffer.from, "function");
+  },
+};
+
+export const workerd_implements_buffer = {
```

Comment:
Not sure if this test has any value as this is already tested in workerd - the infra only runs this code on workerd.

Reply:
I've kept it due to @pi0's request/suggestion.